app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryLabels.js

```diff
@@ -7,7 +7,7 @@ class ExpandedStoryLabels extends React.Component {
     super(props);
 
     this.state = {
-      tags: props.labels
+      labels: props.labels
```

Comment:
As we are using Redux to get and update the project board with their stories and labels, why are we using the internal state here?

Shouldn't be enough just call `onEdit` to update the story and get it updated again from the store?

Reply:
Yes! well thought.